### ISSUE TITLE
Records not available via the ES verification page

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1273]
A verifier points out that he can see records on the 'old' verification page that are not visible in the current ES verification page. He also points out that we no longer have the verification buttons available on the old verification page.

It would be good to switch off the old page, but I am reluctant to do that if we are not confident that records are available on the ES one.

Example 1
Occurrence ID; 12484290
Can be seen on the old verification page but not the new one
Can be seen on the ES Explore page, and the [record details](https://irecord.org.uk/record-details?occurrence_id=12484290) are visible
However, having gone to the record details page, if you then click on the button for "Eplore records ..." button, the [resulting Explore page](https://irecord.org.uk/all-records?filter-date_age=&filter-indexed_location_list=&filter-taxon_meaning_list=243429) is empty
If you click on the species details page the [resulting page](https://irecord.org.uk/species-details?taxon_meaning_id=243429) does show map dots and photos

Example 2
Same as example 1, for occurrence ID 23715311

Examples 1 and 2 both link to taxon "Chromatomyia horticola/syngenesiae agg." (taxa_taxon_list_id = 538741) but I can't see anything unusual in the taxon entry.

Example 3 is for "Phytomyza ranunculella" (taxa_taxon_list_id = 538751).

Example 3
Occurrence ID; 23925470
Can be seen on the old verification page but not the new one
Can be seen on the ES Explore page, and the [record details](https://irecord.org.uk/record-details?occurrence_id=23925470) are visible
However, having gone to the record details page, if you then click on the button for "Eplore records ..." button, the [resulting Explore page](https://irecord.org.uk/all-records?filter-date_age=&filter-indexed_location_list=&filter-taxon_meaning_list=243439) is empty
If you click on the species details page the [resulting page](https://irecord.org.uk/species-details?taxon_meaning_id=243439) does show map dots and photos

**Comment by johnvanbreda** [comment]
These records were in Elasticsearch, but for any taxa added manually to the warehouse, the taxon hierarchy filtering won't work in Elasticsearch unless I update the lookup tables that Logstash uses to populate the index. I've updated the lookups and triggered a refresh of records for recently added taxa.

This is an area that will need some work at some point - completion of automation of the pull from UKSI (which just needs testing) then automating the update of Elasticsearch's lookups.

**Comment by kitenetter** [comment]
@johnvanbreda has "automating the update of Elasticsearch's lookups" been achieved now? If so we can close this.

**Comment by johnvanbreda** [comment]
No, though @burkmarr has made a PHP script that can update the lookups and which could be automated.
